nixos/modules/services/web-apps/mobilizon.nix

```diff
@@ -0,0 +1,438 @@
+{ pkgs, lib, config, ... }:
+
+with lib;
+
+let
+  cfg = config.services.mobilizon;
+
+  user = "mobilizon";
+  group = "mobilizon";
+
+  package = pkgs.mobilizon;
```

Comment:
The convention in Nixpkgs in encountered so far is, that `package` should only be an option if there is actually a second implementation in Nixpkgs. Otherwise overriding via an overlay makes more sense imo.